base/src/main/java/com/epmedu/animeal/base/ui/ShortButton.kt

```diff
@@ -0,0 +1,56 @@
+package com.epmedu.animeal.base.ui
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Divider
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.epmedu.animeal.base.theme.AnimealTheme
+import com.epmedu.animeal.base.theme.DisabledButtonColor
+import com.epmedu.animeal.base.theme.DisabledButtonContentColor
+
+@Composable
+fun ShortButton(
+    text: String,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
```

Comment:
according to google guidlines modifier should be a first parameter

Reply:
Sorry, but documentation says: "It should appear as the first optional parameter in the parameter list; after all required parameters (except for trailing lambda parameters) but before any other parameters with default values.",
text and onClick parameters are required, therefore modifier should be after them.